### ISSUE TITLE
Icon Request: fa-contactless

### ISSUE THREAD

**Issue by stu-care** [FortAwesome/Font-Awesome#9887]
Hello,

Along the lines of apple pay request made previously, can I also request contactless payment logo.

![contactless](https://cloud.githubusercontent.com/assets/22036076/18345953/c80254aa-75b5-11e6-8a6d-944a3bdb0492.png)

Cheers,
Stu

**Comment by xitude** [comment]
+1

**Comment by jacksonj04** [comment]
:+1:, but also the contactless-symbol as described at #8617

**Comment by lazybearcreations** [comment]
Please, please please... This one is a must have.

**Comment by martindubenet** [comment]
I don't get it why, since this request is **closed since 6 months**, that searching for either « `nfc` » or « `contactless` » on [FontAwesome.com's icon search](https://fontawesome.com/icons) does not give me any results!?

Is it included or not in the PRO **version 5.8.1** ?

**Comment by Legenyes** [comment]
+1

**Comment by Legenyes** [comment]
+1

**Comment by VictorBNT** [comment]
+1 Please !

**Comment by martindubenet** [comment]
Still no `fa-nfc`, `fa-contactless` or whatever icon available in version 5.10.1.
I don't get it why!?!?

**#MyTwoCents**
While i'm at it, instead of « contact less » it should be `fa-beacon-tag` AND `fa-beacon-read` since a « beacon » is exactly what it is all about when we compare it to the « barcode » term already used in FontAwesome.

**Comment by Nugerama** [comment]
+1

**Comment by martindubenet** [comment]
+1

**Comment by doceme** [comment]
+1

**Comment by ebreyit** [comment]
I would also like to vote for this one.  Contactless Payment is now big in Europe

**Comment by sachaw** [comment]
+1

**Comment by jfreymann** [comment]
+1

**Comment by antillagorilla** [comment]
I also think this would be a great icon to have. Especially with COVID-19, people are using contactless payments a lot more now.

https://usa.visa.com/run-your-business/accept-visa-payments/posguidelines.html#2
https://www.emvco.com/about/trademark-centre/#menu-item-1406:~:text=EMVCo%20Contactless%20Marks

**Comment by tagliala** [comment]
So this is a brand and this symbol is somehow trademarked?

**Comment by jacksonj04** [comment]
Yes, it's a trademark of EMVCo. But then again pretty much all of the icons in the brands set are also trademarks.

They do make the imagery available for people to use, as long as it's in accordance with their terms (basically, only use it where the service is actually accepted). This is the same as other payment services including Visa, MasterCard, AmEx, PayPal, Google Pay and Apple Pay.

**Comment by tagliala** [comment]
> Yes, it's a trademark of EMVCo.

Thanks

> But then again pretty much all of the icons in the brands set are also trademarks.

Yes, I was asking because it was incorrectly (by me) labeled as "web application icon": if it is a trademark, it should be part of the Brands set and appear in the brands leaderboard

**Comment by copiri-six** [comment]
👍

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No biggie.
